.github/workflows/ci_workflow.yml

```diff
@@ -445,9 +445,140 @@ jobs:
         # The yaml-cpp_VERSION is set below because Findyaml-cpp.cmake needs it but is unable to 
         # extract it from the headers, like the other modules.
         #
-        # Prefer the static version of each dependencies by using <pkg>_STATIC_LIBRARY. 
-        # Alternatively, this can be done by setting <pkg>_LIBRARY and <pkg>_INCLUDE_DIR to 
-        # the static version of the package.       
+        # Prefer the static version of each dependencies by using <pkg>_STATIC_LIBRARY.
+        # Alternatively, this can be done by setting <pkg>_LIBRARY and <pkg>_INCLUDE_DIR to
+        # the static version of the package.
+        run: |
+          cmake . \
+                -DCMAKE_PREFIX_PATH=../../../_install \
+                -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} \
+                -Dexpat_ROOT=${{ env.ocio_build_path }}/ext/dist \
+                -Dexpat_STATIC_LIBRARY=ON \
+                -DImath_ROOT=${{ env.ocio_build_path }}/ext/dist \
+                -DImath_STATIC_LIBRARY=ON \
+                -Dpystring_ROOT=${{ env.ocio_build_path }}/ext/dist \
+                -Dyaml-cpp_ROOT=${{ env.ocio_build_path }}/ext/dist \
+                -Dyaml-cpp_STATIC_LIBRARY=ON \
+                -Dyaml-cpp_VERSION=0.7.0 \
+                -DZLIB_ROOT=${{ env.ocio_build_path }}/ext/dist \
+                -DZLIB_STATIC_LIBRARY=ON \
+                -Dminizip-ng_ROOT=${{ env.ocio_build_path }}/ext/dist \
+                -Dminizip-ng_STATIC_LIBRARY=ON
+          cmake --build . \
+                --config ${{ matrix.build-type }}
+          ./consumer
+        working-directory: _build/tests/cmake-consumer-dist
+
+  # ---------------------------------------------------------------------------
+  # macOS Arm
+  # ---------------------------------------------------------------------------
+
+  macos-arm:
+    name: 'macOS 14 arm
+      <AppleClang
+        arch=${{ matrix.arch-type }},
+        config=${{ matrix.build-type }},
+        shared=${{ matrix.build-shared }},
+        simd=${{ matrix.use-simd }},
+        cxx=${{ matrix.cxx-standard }},
+        python=${{ matrix.python-version }},
+        docs=${{ matrix.build-docs }},
+        oiio=${{ matrix.use-oiio}}>'
+
+    runs-on: macos-14
+    strategy:
+      matrix:
+        build: [1, 2]
+        include:
+          - build: 1
+            arch-type: "arm64"
+            test-rosseta: "OFF"
```

Comment:
rosseta --> rosetta

Reply:
haha, you have no idea how many times I've misspelled that!